students/WesleyM/Lesson3/ListLab.py

```diff
@@ -0,0 +1,79 @@
+def fruit1():
+    """
+    List four fruits, includes a user input to list,
+    and removes fruit that start with the letter "P"
+    """
+    fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+
+    print(fruit)
+    
+    response = input("Input a fruit > ")
+
+    fruit.append(response)
+
+    print(fruit)
+  
+    numres = int(input("Input a number > "))
+    while numres < 1 or numres > 5:
+        print('Input a number between 1 and 5')
+        numres = int(input("Input a number > "))
+    print(fruit[numres-1])
+    response = input("Input another fruit > ")
+    fruit = [response] + fruit
+    print(fruit)
+    response = input("Input another fruit > ")
+    fruit.insert(0, response)
+
+    for i in fruit:
+        if i[0] == ('P'):
```

Comment:
You could use the `lower()` string method here to catch upper and lower case 'p's: `if i[0].lower() == 'p':`. Also no need for the parentheses around the `'P'`